master/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -61,19 +54,19 @@ associated with that network.
 #### Create a Docker network
 
 To create a Docker network using Calico, run the `docker network create`
-command specifying "calico" as both the network and IPAM driver.
+command specifying `calico` as the network driver and `calico-ipam` as the IPAM driver.
 
 For example, suppose we want to provide network policy for a set of database
 containers.  We can create a network called `databases` with the the following
 command:
 
 ```
-docker network create --driver calico --ipam-driver calico databases 
+docker network create --driver calico --ipam-driver calico-ipam databases 
 ```
 
 #### View the policy associated with the network
 
-You can use the `calicoctl profile <profile> rule show` to display the
+You can use the `calicoctl get -o yaml profile <profile>` to display the
```

Comment:
I don't think this will work as per the timeline in these instructions.  You need to create an endpoint first, or just create the profile in advance.